plugin/showpass-wordpress-plugin-shortcode.php

```diff
@@ -856,6 +864,18 @@ function showpass_widget_expand($atts, $content = null) {
 			$button .= sprintf('data-tracking="%s" ', $tracking);
 		}
 
+    if ($show_description) {
+      $button .= sprintf('data-show-description="%s" ', $show_description);
+    }
+
+    if ($keep_shopping) {
+      $button .= sprintf('data-shopping="%s" ', $keep_shopping);
+    }
+
+    if ($show_specific_tickets) {
+      $button .= sprintf('data-show-specific-tickets="%s" ', $show_specific_tickets);
```

Comment:
I forget how this works - where does the` data-show-specific-tickets` value get passed to the API to only return a subset tickets?